### ISSUE TITLE
[terra-heading] Missing themeable variables

### ISSUE THREAD

**Issue by ryanthemanuel** [cerner/terra-core#3050]
# Bug Report

The following themeable variables are missing:

**clinical-lowlight-theme**

--terra-heading-attention-color - packages/terra-heading/lib/terra-dev-site/doc/example/colors.module.scss (line: 14, column: 4)
--terra-heading-default-color - packages/terra-heading/lib/terra-dev-site/doc/example/colors.module.scss (line: 10, column: 4)
--terra-heading-info-color - packages/terra-heading/lib/terra-dev-site/doc/example/colors.module.scss (line: 22, column: 4)
--terra-heading-success-color - packages/terra-heading/lib/terra-dev-site/doc/example/colors.module.scss (line: 18, column: 4)

**orion-fusion-theme**

--terra-heading-attention-color - packages/terra-heading/lib/terra-dev-site/doc/example/colors.module.scss (line: 14, column: 4)
--terra-heading-default-color - packages/terra-heading/lib/terra-dev-site/doc/example/colors.module.scss (line: 10, column: 4)
--terra-heading-info-color - packages/terra-heading/lib/terra-dev-site/doc/example/colors.module.scss (line: 22, column: 4)
--terra-heading-success-color - packages/terra-heading/lib/terra-dev-site/doc/example/colors.module.scss (line: 18, column: 4)

**Comment by neilpfeiffer** [comment]
For the Lowlight theme:
```
:local {
  .clinical-lowlight-theme {
    ...

    // Used for docsite custom color examples only 
    --terra-heading-attention-color: #fb4c4c;
    --terra-heading-default-color: #6f7477;
    --terra-heading-info-color: #5a81ff;
    --terra-heading-success-color: #78c346;
  }
}
```

----

For the Fusion theme:
```
:local {
  .orion-fusion-theme {
    ...

    // Used for docsite custom color examples only 
    --terra-heading-attention-color: #c00;
    --terra-heading-default-color: #1c1f21;
    --terra-heading-info-color: #007cc3;
    --terra-heading-success-color: #2e7d32;
  }
}
```

**Comment by emilyrohrbough** [comment]
Resolved in https://github.com/cerner/terra-core/pull/3107
